api/database/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.3'
```

Comment:
All of this should be part of the main docker-compose.yml file. The goal is to have dashboard, api, and db all running together.

Reply:
updated to the main docker-compose.yml file